tests/mothership/test_collateral.py

```diff
@@ -0,0 +1,160 @@
+    assert mothership.collateralActive(collateral) is False
+    assert mothership.totalCollaterals() == total
+
+    assert token.hasRole(token.MINTER_ROLE(), collateral) is False
+    assert token.hasRole(token.BURNER_ROLE(), collateral) is False
+
+
+def test_initialize_collateral_reverts_when_exists(mothership, collateral,
+                                                   gov):
+    _ = mothership.initializeCollateral(collateral, {"from": gov})
+
+    EXPECTED_ERROR_MESSAGE = 'OVLV1: collateral exists'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.initializeCollateral(collateral, {"from": gov})
+
+
+def test_initialize_collateral_reverts_when_not_gov(mothership, collateral,
+                                                    bob):
+    EXPECTED_ERROR_MESSAGE = 'OVLV1:!gov'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.initializeCollateral(collateral, {"from": bob})
+
+
+def test_enable_collateral_reverts_when_not_initialized(mothership, collateral,
+                                                        gov):
+    EXPECTED_ERROR_MESSAGE = 'OVLV1: collateral !exists'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.enableCollateral(collateral, {"from": gov})
+
+
+def test_enable_collateral_reverts_when_not_disabled(mothership, collateral,
+                                                     gov):
+    _ = mothership.initializeCollateral(collateral, {"from": gov})
+
+    EXPECTED_ERROR_MESSAGE = 'OVLV1: collateral !disabled'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.enableCollateral(collateral, {"from": gov})
+
+
+def test_enable_collateral_reverts_when_not_gov(mothership, collateral, gov,
+                                                bob):
+    _ = mothership.initializeCollateral(collateral, {"from": gov})
+    _ = mothership.disableCollateral(collateral, {"from": gov})
+
+    EXPECTED_ERROR_MESSAGE = 'OVLV1:!gov'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.enableCollateral(collateral, {"from": bob})
+
+
+def test_disable_collateral_reverts_when_not_initialized(mothership,
+                                                         collateral,
+                                                         gov):
+    EXPECTED_ERROR_MESSAGE = 'OVLV1: collateral !exists'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.disableCollateral(collateral, {"from": gov})
+
+
+def test_disable_collateral_reverts_when_not_enabled(mothership, collateral,
+                                                     gov):
+    _ = mothership.initializeCollateral(collateral, {"from": gov})
+    _ = mothership.disableCollateral(collateral, {"from": gov})
+
+    EXPECTED_ERROR_MESSAGE = 'OVLV1: collateral !enabled'
+    with brownie.reverts(EXPECTED_ERROR_MESSAGE):
+        mothership.disableCollateral(collateral, {"from": gov})
+
+
+def test_disable_collateral_reverts_when_not_gov(mothership, collateral, gov,
```

Comment:
Here is a function we might also want to have a guardian role authorized for. If something is going haywire, we need immediate action.